src/test/java/teammates/test/cases/ui/browsertests/InstructorEditStudentFeedbackPageUiTest.java

```diff
@@ -35,11 +36,25 @@ public static void classSetup() {
     
     @Test
     public void testAll() throws Exception {
+        testModerationHint();
         testEditResponse();
         testAddResponse();
         testDeleteResponse();
     }
     
+    private void testModerationHint() throws Exception {
+        ______TS("verify moderation hint");
+        
+        submitPage = loginToInstructorEditStudentFeedbackPage("IESFPTCourseinstr", "student1InIESFPTCourse@gmail.tmt",
+                "session1InIESFPTCourse");
+        
+        submitPage.verifyHtmlPart(By.className("navbar"), "/instructorEditStudentFeedbackHintLess.html");
+        submitPage.click(By.id("moderationHintButton"));
```

Comment:
Sorry I "👍"ed  the wrong comment. 

> A more appropriate approach would be creating another page `InstructorEditStudentFeedbackPage` which inherit from `FeedbackSubmitPage` and put the method there. But this could be another PR.

This approach sounds good.